cfpb/cfgov-refresh
# reuse Ask CFPB export data script to get enforcement actions data

adds a GUI to Wagtail Admin so enforcement content editors can download a spreadsheet of their data.

## Additions

- export script for enforcement actions pages, adapted from the Ask CFPB export script and from code by @willbarton 
- wagtail hooks and admin template for downloading the script from the Wagtail admin, also based on the "Export Ask data" page in the admin

## Removals

-

## Changes

-

## Testing

1. run this branch locally
1. log in to Wagtail admin locally http://localhost:8000/admin/
1. Go to http://localhost:8000/admin/export-enforcement-actions or click on the link in the sidebar:
![Screen Shot 2019-10-11 at 4 12 41 PM](https://user-images.githubusercontent.com/702526/66682047-0822b600-ec42-11e9-9439-c79f1c2a2fe6.png)
1. Open the CSV in Excel or a text editor and confirm the following fields are all present, and that there are 231 rows (including the header row)

Matter name | Date filed | URL | Status | Category | File number | Content | Preview text

## Screenshots
![Screen Shot 2019-10-11 at 4 17 21 PM](https://user-images.githubusercontent.com/702526/66682250-a0b93600-ec42-11e9-98b6-f86946b5b26e.png)



## Notes

- 

## Todos

- Think about a better way to name the link in the sidebar
- Future PRs: should we consolidate the Ask data & Enforcement actions data exports into one export page in the admin? and then some type of selection of which data to export appears there? shall we implement an Wordpress-export-data style interface? I need to get UX ideas and admin possibilities from @ielerol and @virginiacc

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
